docs/intree-migrate.md

```diff
@@ -0,0 +1,148 @@
+# In-tree storage plugin to CSI Driver Migration
+
+This document covers the example usage of in-tree RBD storage plugin to CSI
+migration feature which can be enabled in a Kubernetes cluster. At present, this
+feature is only supported for RBD in-tree plugin. Once this feature is enabled,
+the in-tree volume requests (`kubernetes.io/rbd`) will be redirected to a
+corresponding CSI (`rbd.csi.ceph.com`) driver.
+
+## RBD
+
+- [Prerequisite](#prerequisite)
+- [Volume operations after enabling CSI migration](#volume-operations-after-enabling-csi-migration)
+  - [Create volume](#create-volume)
+  - [Mount volume to a POD](#mount-volume-to-a-pod)
+  - [Resize volume](#resize-volume)
+  - [Unmount volume](#unmount-volume)
+  - [Delete volume](#delete-volume)
+- [References](#additional-references)
+
+### Prerequisite
+
+For in-tree RBD migration to CSI driver to be supported for your Kubernetes
+cluster, the Kubernetes version running in your cluster should be >= v1.23. We
+also need sidecar controllers (`csi-provisioner` and `csi-resizer`) which are
+compatible with the Kubernetes version v1.23 to be available in your cluster.
+You can enable the migration with a couple of feature gates in your Kubernetes
+cluster. These feature gates are alpha in Kubernetes 1.23 release.
+
+- `CSIMigrationRBD`: when enabled, it will redirect traffic from in-tree rbd
+  plugin (`kubernetes.io/rbd`)  to CSI driver (`rbd.csi.ceph.com`) , default
+  to `false` now.
+- `IntreePluginRBDUnregister`: Disables the RBD in-tree driver
+
+To enable feature gates,
+refer [feature gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/)
+
+As a Kubernetes cluster operator that administers storage, here are the
```

Comment:
`As a Kubernetes cluster operator that administers storage` can be written as `As a Kubernetes storage administrator`?